projects/fsastorefrontlib/src/cms-components/my-account/policy/policy.module.ts

```diff
@@ -68,6 +68,10 @@ const routes: Routes = [
         },
       },
     }),
+    /**
+     * Preparation for final solution for lazy loaded module in libs.
```

Comment:
Put TODO prefix so we can easily search for all comments that need to be removed before release or need some coding effort